manim/camera/camera.py

```diff
@@ -78,6 +72,25 @@ def __init__(self, background=None, **kwargs):
             Any local variables to be set.
         """
         digest_config(self, kwargs, locals())
+
+        # All of the following must be set to the value in stored in config at
+        # the time of _instance construction_.  Before, they were in the CONFIG
+        # dict, which is a class attribute and is defined at the time of _class
+        # definition_.
+        for attr in [
```

Comment:
**Question** : So, if I understand correctly, you said that now this uses `kwargs` in addition to the config dict for these attributes, doesn't it? 
If yes, I think you should add a quick line that would explain that, because your comment is very good but ... I didn't get it immediately.

Reply:
Done in latest commit